recipes/libcurl/all/conanfile.py

```diff
@@ -595,9 +604,9 @@ def package(self):
             rm(self, "*.la", os.path.join(self.package_folder, "lib"))
             if self._is_mingw and self.options.shared:
                 # Handle only mingw libs
-                copy(self, pattern="*.dll", src=self.build_folder, dst="bin", keep_path=False)
-                copy(self, pattern="*.dll.a", src=self.build_folder, dst="lib", keep_path=False)
-                copy(self, pattern="*.lib", src=self.build_folder, dst="lib", keep_path=False)
+                copy(self, pattern="*.dll", src=self.build_folder, dst=os.path.join(self.package_folder, "bin"), keep_path=False)
+                copy(self, pattern="*.dll.a", src=self.build_folder, dst=os.path.join(self.package_folder, "lib"), keep_path=False)
+                copy(self, pattern="*.lib", src=self.build_folder, dst=os.path.join(self.package_folder, "lib"), keep_path=False)
```

Comment:
good catch, I think it will close https://github.com/conan-io/conan-center-index/issues/14404

/cc @klausholstjacobsen